packages/cli/src/daemon-config.ts

```diff
@@ -153,6 +153,12 @@ export class DaemonHTTPApiConfig {
     },
   })
   corsAllowedOrigins?: RegExp[]
+
+  /**
+   * An array of DIDs with access to Admin API (represented as strings)
+   */
+  @jsonArrayMember(String, { name: 'admin-dids' })
```

Comment:
note this is a breaking change to the config file format (but only since this functionality went in last week)

Now that the Ceramic core implementation actually does no enforcement of access control on the admin api it made more sense to me that this option be considered part of the http-api config instead of the node config section of the config file